etl/steps/data/garden/wb/2024-09-09/food_prices_for_nutrition.meta.yml

```diff
@@ -0,0 +1,325 @@
+definitions:
+  description_key_pct_of_population: &description_key_pct_of_population
+    # Original description:
+    # The indicator expresses the percentage of the total population unable to afford an energy sufficient diet in a given country and year. A diet is considered unaffordable when the diet cost plus expenditures for basic non-food needs exceed income per capita per day. The expenditures for basic non-food needs are calculated as average non-food expenditure shares of low-income consumer multipled by internaitonal poverty lines set by the World Bank. The non-food expenditure share is 37% and 44% in low-income and lower-middle-income countries for the second quintile of consumers, and 54% in upper-middle-and-high-income countries for the first quintile of consumers, according to household surveys compiled by the World Bank. The international poverty lines are $2.15/day for low-income countries, $3.65/day for lower-middle-income countries, $6.85/day for upper-middle-income countries, and $24.36/day for high-income countries, in 2017PPP$. Countries' income classifications follow the calendar year of 2021 standard (fiscal year of 2023 of the World Bank), which is the base year of the latest ICP cycle. Income data are provided by the World Bank’s Poverty and Inequality Platform. A value of zero indicates a null or a small number rounded down at the current precision level.
+    - Non-food expenses are estimated by looking at how much low-income people typically spend on things like housing, clothing, and transportation.
+    - "This amount is calculated as: [Average share of non-food expenditure for a low-income consumer] x [International poverty line]."
+    - "The average share of non-food expenditure is based on the spending habits of specific income groups (quintiles):
+      - In upper-middle-income and high-income countries, the first quintile of consumers is considered (the poorest 20% of consumers), where people spend around 54% of their income on non-food needs.
+      - In low-income and lower-middle-income countries, the second quintile is considered (the second lowest 20% of consumers), where people spend about 37% and 44% of their income on non-food needs, respectively."
+    - "The international poverty lines (in international dollars based on 2017 prices) are:
```

Comment:
I agree, but I adapted these definitions from [the World Bank metadata](https://databank.worldbank.org/source/food-prices-for-nutrition#selectedDimension_DBList) (click on the (i) button of, e.g. Series -> "Percent of the population who cannot afford sufficient calories"). Do you have any suggestion for how to call them otherwise?

Reply:
For this dataset they call it like that, so I would keep it. I was just saying this as a side note. I don't think they have a collective name outside this dataset.